From 7bc2645af5e7e638cbd6986cf707507f3d54cc3b Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Fri, 28 Dec 2018 17:22:57 +0800
Subject: [PATCH 026/235] ARM: dts: ls1021aqds: Enable the esdhc

This patch is to set the esdhc's status to "okay" in kernel.

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a-qds.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/ls1021a-qds.dts b/arch/arm/boot/dts/ls1021a-qds.dts
index da4328c27d20..980fedfefb69 100644
--- a/arch/arm/boot/dts/ls1021a-qds.dts
+++ b/arch/arm/boot/dts/ls1021a-qds.dts
@@ -336,6 +336,10 @@
 	};
 };
 
+&esdhc {
+	status = "okay";
+};
+
 &sai2 {
 	status = "okay";
 };
-- 
2.17.1

